back/spec/requests/rack_attack_spec.rb

```diff
@@ -0,0 +1,307 @@
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+describe 'Rack::Attack', type: :request do
```

Comment:
As these tests are so slow (plus, this code is unlikely to change), it could make sense to add `slow_test: true`, so they will run only in master and production.

```ruby
describe 'Rack::Attack', type: :request, slow_test: true do
```

And then you will run them like that
```ruby
docker-compose run web bundle exec rspec ./engines/ee/multi_tenancy/spec/services/tenant_template_service_spec.rb -t slow_test
```

Reply:
On the other hand, as `slow_tests` are run in just one container (I suppose we can/should improve it), it could make sense to add some tests that send 10 and more requests to `xit` section.

Reply:
Yes, I agree. I think it's probably best to reduce/avoid adding tests to the 'slow tests' CI unless really necessary.

Although there are _**no**_ tests here that send less than 10 requests, since there are no rules that act on less than 10 requests.

How about I just keep the 2 fastest tests active (less than 2 seconds to run in local dev env)?

Reply:
Or you meant keep all tests _with 10 or less_ requests active?

Reply:
No, I meant we could disable the slowest tests (e.g. with `xit`).
And I still think it's better to put them to `slow_test` because this way we would clearly see what tests are slow. Also, sooner or later we will parallelize slow tests.

Reply:
But in general, it's not that critical.